### DISCUSSION TITLE
Welcome to atom Discussions!

### DISCUSSION THREAD

**Discussion by aellispierce** [atom/atom#22020] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by calebmeyer** [comment]
Hi, I'm Caleb. I have been active on the atom slack and discord for a long time, and written a few packages with varying levels of success. 

I've always loved the atom project because I think it perfectly straddles the line between usable out of the box and hackable to the core. Also all the core devs I've interacted with have been really helpful and friendly.

**Comment by UziTech** [comment]
Hi, I'm Tony

I am a maintainer of:

- [Atom Community](https://github.com/atom-community)
- [Atom Linter](https://github.com/AtomLinter)
- [Atom Format](https://github.com/AtomFormat)
- [Bus Stop](https://github.com/bus-stop) (Atom Terminals)
- [Atom Minimap](https://github.com/atom-minimap)
- [And many more Atom packages](https://atom.io/users/UziTech/packages)

I second @calebmeyer that the hackability and usability of Atom is what makes it amazing.

**Reply by idleberg** [reply]
I've been developing for Atom since May 2014, which makes me a proud owner of those sew-on patches GitHub mailed out back in the day. My relationship with Atom is a bit unusual, I guess: I love it as a platform, I haven't seen such a powerful API in any of the other hip editors out there, but I don't use it for development myself. One of the most underappreciated things about Atom is probably its looks, it's beautiful out of the box.

Other than that, I maintain the [@allaboutatom](https://twitter.com/allaboutatom) bot on Twitter, mainly to keep myself updated. Speaking of which, I wonder how to update it in the future, since there doesn't seem to be an Atom (as in the syndication format) feed for discussions.

**Reply by ThatXliner** [reply]
I can do some webscraping and maybe whip up an unofficial API. Who knows? Maybe GitHub will add a feed for it soon...

**Reply by idleberg** [reply]
https://docs.github.com/en/rest/reference/teams#discussions

**Reply by ThatXliner** [reply]
Oop- nevermind then!

**Reply by UziTech** [reply]
Are team discussions the same as repo discussions in octokit?

I'm not sure what the `team_slug` would be in the url.
`/orgs/{org}/teams/{team_slug}/discussions`
